src/revealer/web/index.js

```diff
@@ -0,0 +1,32 @@
+import { getIframeId, sendMessage, onViewport } from '../../_shared/js/messages.js';
+
+getIframeId()
+.then(() => {
+    onViewport(({ height }) => {
+        const [ scrollType,
+            backgroundColour,
+            backgroundImage,
+            backgroundRepeat,
+            backgroundPosition,
+            backgroundSize ] = [
+            'fixed',
+            '#f6f6f6',
+            `url('[%BackgroundImage%]')`,
+            'no-repeat',
+            'center center',
+            'cover'
+        ];
+
+        // for the height, we need to account for the height of the location bar, which
+        // may or may not be there. 70px padding is not too much.
+        sendMessage('background', {
+            scrollType,
+            backgroundColour,
+            backgroundImage,
+            backgroundRepeat,
+            backgroundPosition,
+            backgroundSize,
+            height: `${height + 70}px`
```

Comment:
It's an offset to make the image go further, not a box model sizing. Comes from here: https://github.com/guardian/frontend/blob/9eecc84ac32967f23a938908e0645d0b11efcd5d/static/src/javascripts/projects/commercial/modules/creatives/revealer.js#L62